### PR TITLE
fix: 🐛 all duplicate routes

### PR DESCRIPTION
**Pull request by rsteubs**
### Feature
- [x] Updated CHANGELOG.md
- [x] Link to issue [Link](https://app.asana.com/0/1163629870679883/1200594450068003)
- [x] Add PR label

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,11 +65,12 @@
     "highcharts": "9.1.2",
     "highcharts-vue": "1.4.0",
     "idna-uts46": "1.1.0",
+    "lodash": "4.17.21",
```
Comment (gamalielhere) [inline package.json:68]: web3 actually comes with underscore. you can remove this and use something similar from them instead.

File: src/modules/balance/ModuleTokens.vue [hunk 2]
```diff
@@ -54,6 +54,8 @@
 import { mapGetters, mapState } from 'vuex';
 import BalanceEmptyBlock from './components/BalanceEmptyBlock';
 import { formatFiatValue } from '@/core/helpers/numberFormatHelper';
+import isEqual from 'lodash/isEqual';
```
Comment (gamalielhere) [inline src/modules/balance/ModuleTokens.vue:57]: replace with `import { _ } from 'web3-utils'; _.isEqual works the same

File: src/modules/balance/ModuleTokens.vue [hunk 3]
```diff
@@ -142,7 +144,9 @@ export default {
                   toToken: '',
                   amount: item.balancef
                 };
-                this.$router.push({ name: 'Swap', query: obj });
+                if (!isEqual(obj, this.$route.query)) {
```
Comment (gamalielhere) [inline src/modules/balance/ModuleTokens.vue:147]: use `_.isEqual`

File: src/modules/create-wallet/ModuleCreateWalletSoftware.vue [hunk 4]
```diff
@@ -139,12 +139,12 @@ export default {
      * @type - must be one of the this.types or an empty string (this will reset step to 0)
      */
     setType(newType) {
-      try {
```
Comment (gamalielhere) [inline src/modules/create-wallet/ModuleCreateWalletSoftware.vue:142]: you can probably revert this once you merge the other PR with the Router config change.

**Review by gamalielhere — CHANGES_REQUESTED** [review]

**Review by gamalielhere — APPROVED** [review]

File: src/core/router/index.js [hunk 5]
```diff
@@ -34,11 +34,7 @@ router.beforeResolve((to, from, next) => {
   } else {
     if (store.state.wallet.address === null) {
       store.dispatch('external/setLastPath', to.path);
-      /** only resolve a new path when current path isn't accesswallet
-       * this is a little hacky but utilizes intended purpose.
-       * vue router notes: move on to the next hook in the pipeline. If no hooks are left, the navigation is confirmed.
-       */
-      if (from.name !== 'AccessWallet') router.push({ name: 'AccessWallet' });
+      next({ name: 'AccessWallet' });
```
Comment (kvhnuke) [inline src/core/router/index.js:37]: merge the routes branch with this and replace `AccessWallet` with the right route

**Review by kvhnuke — CHANGES_REQUESTED** [review]

**Review by kvhnuke — APPROVED** [review]

**Pull request merged**